tst/working/quick/PermLargeBasePrimitive.tst

```diff
@@ -1,13 +1,29 @@
-#@local S19OnSets, jellyGroup, n, random, jellyGroupConj, ri
+#@local S17OnSets, jellyGroup, n, random, jellyGroupConj, ri
+#@local addFixedPointsPerm, jellyGroupWithFixedPoints, riWithFixed
 
 # Test for LargeBasePrimitive
-gap> S19OnSets := Action(SymmetricGroup(19), Combinations([1 .. 19], 2), OnSets);;
-gap> jellyGroup := WreathProductProductAction(S19OnSets, Group((1,2)));;
+gap> S17OnSets := Action(SymmetricGroup(17), Combinations([1 .. 17], 2), OnSets);;
+gap> jellyGroup := WreathProductProductAction(S17OnSets, Group((1,2)));;
 gap> n := NrMovedPoints(jellyGroup);;
 gap> # Take a random conjugate
 gap> random := Random(SymmetricGroup(n));;
 gap> jellyGroupConj := jellyGroup ^ random;;
-gap> RECOG.TestGroup(jellyGroupConj,
+gap> ri := RECOG.TestGroup(jellyGroupConj,
 >   false,
 >   Size(jellyGroup),
 >   rec(tryNonGroupElements := true));;
+gap> ri!.fhmethsel.successMethod;
+"LargeBasePrimitive"
+
+# Test that groups with fixed points are handled together by
+# ThrowAwayFixedPoints and LargeBasePrimitive.
```

Comment:
Maybe reference this PR and/or issue #190 in the comment, to make it easy to get further details if required?

Reply:
Done.